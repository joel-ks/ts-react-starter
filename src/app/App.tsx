import * as React from "react";
import {hot} from "react-hot-loader/root";

const App: React.FC = () => (
    <p>Working...</p>
);

export default hot(App);
