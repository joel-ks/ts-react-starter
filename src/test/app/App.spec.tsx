import * as React from "react";
import { render } from "@testing-library/react";
import App from "../../app/App";


test("App.tsx renders as expected", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});
