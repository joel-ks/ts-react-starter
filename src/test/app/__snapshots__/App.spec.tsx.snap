// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.tsx renders as expected 1`] = `
<DocumentFragment>
  <p>
    Working...
  </p>
</DocumentFragment>
`;
